doc/source/user-environment.rst

```diff
@@ -219,27 +221,54 @@ variables.
 Pre-populating user's ``$HOME`` directory with notebooks
 --------------------------------------------------------
 
-By default, the contents of ``$HOME`` in the docker image are hidden by
-the contents of the per-user persistent volume. If you want to, you can
-execute a command before the notebook starts each time and copy the files
-you want from your image to the user's home directory.
+If persistent storage is enabled (on by default) then the contents of
+``$HOME`` in the docker image will be hidden to the user. In order to
```

Comment:
When persistent storage is enabled (which is the default), contents of the docker image's ``$HOME`` directory will be hidden from the user. To provide these contents and make them visible to the user, you must pre-populate the user's filesystem. To do so, you would include commands in the ``config.yaml`` that would be run each time a user starts their server. The following pattern can be used in ``config.yaml``: